snp_blending/config/workcell_plugins.yaml

```diff
@@ -0,0 +1,34 @@
+kinematic_plugins:
+  search_libraries:
+    - snp_motion_planning_plugins
```

Comment:
I don't think we have libraries in which to search for custom kinematics plugins

```suggestion
  search_libraries: []
```